gazebo_ros2_control/src/gazebo_system.cpp

```diff
@@ -27,6 +27,8 @@
 #include "hardware_interface/hardware_info.hpp"
 #include "hardware_interface/types/hardware_interface_type_values.hpp"
 
+using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```

Comment:
```suggestion
using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```


is this needed?

Reply:
It should be needed, but probably under "namespace" scope and not global scope.